File: java/Tife Assignment/AgeDifference.java
import java.util.Scanner;

public class AgeDifference{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

System.out.print("Enter father's age (1-80): ");
int currentFatherAge = input.nextInt();

System.out.print("Enter son's age (1-80): ");
int currentSonAge = input.nextInt();

if(currentFatherAge < 1 || currentFatherAge > 80 || currentSonAge < 1 || currentSonAge > 80) {
System.out.println("Invalid enter age between (1-80)");
return;
}

int ageDifference = Math.abs(currentFatherAge - 2 * currentSonAge);
System.out.println(ageDifference);

}    
}

File: python/Ebuka-Assignment/age_difference.py
"""
1. collect two input fathers and son's age
2. calcucalte the age difference and store in a variable
3. check if age is greater than 80 or less than one
4. print result

"""


father_age = int(input("ENter father age between (1-80)"));

son_age = int(input("ENter son age between (1-80)"));

age_difference = (father_age - 2 * son_age);

if(father_age < 1 or father_age > 80 or son_age < 1 or son_age > 80):
	print("Invalid enter age between (1-80)");



print(f"the age difference between father and son is {age_difference}");